tests/fsharp/SDKTests/tests/DefaultFSharpCore - IsPreviewSdkEnablePreviewCheck.proj

```diff
@@ -0,0 +1,21 @@
+<Project ToolsVersion="4.0" DefaultTargets="Test" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Verifies that in a preview sdk with preview check enabled it selects the shipped fsharp.core package -->
+
+  <PropertyGroup>
+    <_NETCoreSdkIsPreview>true</_NETCoreSdkIsPreview>
+  </PropertyGroup>
+
+  <Import Project="Test.props" />
+
+  <PropertyGroup>
+    <ExpectsValueTuple>false</ExpectsValueTuple>
+    <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
+    <DisableFSharpCorePreviewCheck>false</DisableFSharpCorePreviewCheck>
+    <ExpectsFSharpCore>true</ExpectsFSharpCore>
+    <ExpectedFSharpCorePackageVersion>$(FSharpCoreShippedPackageVersion)</ExpectedFSharpCorePackageVersion>
```

Comment:
Is preview = true and enable preview check = true, shouldn't this be $(FSCorePackageVersion)?  E.g., the not-yet-shipped one?